website/content/docs/connect/proxies/envoy-extensions/configuration/otel-access-logging.mdx

```diff
@@ -0,0 +1,390 @@
+
+- Default: `1s`
+- Data type: String
+
+### `Arguments.Config.BufferSizeBytes`
+
+Specifies soft size limit in bytes for access log entries buffer. Logger will buffer requests until this limit it hit, or every time flush interval is elapsed, whichever comes first. Setting it to zero effectively disables the batching.
+
+#### Values
+
+- Default: `16384`
+- Data type: Integer
+
+### `Arguments.Config.FilterStateObjectsToLog`
+
+Specifies additional filter state objects to log in filter_state_objects. Logger will call FilterState::Object::serializeAsProto to serialize the filter state object.
+
+#### Values
+
+- Default: None
+- Data type: List of String
+
+### `Arguments.Config.RetryPolicy`
+
+Defines a policy for retrying requests to the upstream service when fetching the plugin data. The `RetryPolicy` field is a map containing the following parameters:
+
+- [`RetryBackoff`](#pluginconfig-vmconfig-code-remote-retrypolicy)
+- [`NumRetries`](#pluginconfig-vmconfig-code-remote-numretries)
+
+#### Values
+
+- Default: None
+- Data type: Map
+
+### `Arguments.Config.RetryPolicy.RetryBackOff`
+
+Specifies parameters that control retry backoff strategy. 
+
+#### Values
+
+- Default: None
+- Data type: Map
+
+The following table describes the fields you can specify in the `RetryBackOff` map:
+
+| Parameter      | Description                                                                                                                                                | Data type | Default |
+| -------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------- | --------- | ------- |
+| `BaseInterval` | Specifies the base interval for determining the next backoff computation. Set a value greater than `0` and less than or equal to the `MaxInterval` value.  | String    | `1s`    |
+| `MaxInterval`  | Specifies the maximum interval between retries. Set the value greater than or equal to the `BaseInterval` value.                                           | String    | `10s`   |
+
+### `Arguments.Config.RetryPolicy.NumRetries`
+
+Specifies the number of times Envoy retries to fetch plugin data if the initial attempt is unsuccessful.
+
+#### Values
+
+- Default: `1`
+- Data type: Integer
+
+### `Arguments.Config.Body`
+
+Specifies OpenTelemetry [LogResource](https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/logs/v1/logs.proto) fields, following [Envoy access logging formatting](https://www.envoyproxy.io/docs/envoy/latest/configuration/observability/access_log/usage). See ‘body’ in the LogResource proto for more details.
+
+#### Values
+
+- Default: None
+- Data type: String
+
+### `Arguments.Config.Attributes`
+
+Specifies `attributes` in the [LogResource](https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/logs/v1/logs.proto). See ‘attributes’ in the LogResource proto for more details.
```

Comment:
```suggestion
Specifies `attributes` in the [LogResource](https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/logs/v1/logs.proto). Refer to `attributes` in the LogResource proto for more details.
```
I'm a little unclear on what we're referring people to in the second sentence. Are we telling people to check an entry called `attributes` in some output? Would this be clearer to the target audience?